meetings/Agenda_2017-05-03.md

```diff
@@ -0,0 +1,41 @@
+## Meeting Agenda, May 3, 2017
+
+1. Review 
+ * Attendance surveys - are we adding people to email list? Have we / should we send followup emails to new people? 
+ * We now have a kettle - need cups!
+ * New group project proposed by Lina - brief description for group
+
+2. Summer plans
+ * Survey results - link shared with council
+ * Code-alongs are most popular format followed by lightning demos and discussion sessions
+ * Proposed summer topics based on survey results (and also what I want):
+	* Collaborating with others using GitHub
+	* Git: continuous integration and testing
+	* Python: SciKit learn - regression, fitting, machine learning
+	* Python: unit test pair programming session
+	* Python: intro to GPU programming
+	* Themed coworking for packages, both R and Python - help people (cough me cough) actually do it
+	* R: dplyr and magrittr
+	* Intro to Bash
+	* HTML and/or Java and/or CSS
+	* Docker
+	* Data visualization lightning demos
+ * Advertising push? 
+
+3. Ryerson SWC coming up: reminder
+ * Reconnect with folks there to discuss supplies, helpers, etc.
+
+4. Workshop for DHN
+ * Dates: May 23-24 (three weeks away!)
+ * Potential instructors: Joel, Derek, Dano, Madeleine, Raul
+ * Proposed topics given instructor preferences:
+	* HTML - Dano / Raul
+	* Intro Python and image processing - Joel / Derek
+	* Signal processing and analysis of sound - Dano
+	* Bash: automating stuff, searching for stuff, regular expressions, possibly Git - Madeleine / Joel / Raul
+	* Maybe a session of shorter demos - LaTeX, pandoc, metadata, etc.? 
+
+5. RQM
+ * New draft of syllabus to Martin
+ * Discuss assessment 
+
```

Comment:
Let's also find a few times next week that works for most of us to go and inspect the lecture halls.